app/assets/stylesheets/bootstrap-variables.scss

```diff
@@ -20,7 +20,7 @@ $brand-primary: darken(#0097FF,5%) !default; // darker creation-blue
 $brand-success: #8EDE3D !default;
 // $brand-info:            #5bc0de
 // $brand-warning:         #f0ad4e
-// $brand-danger:          #d9534f
+$brand-danger: #A80000;
```

Comment:
**before**:
![red button before](https://cloud.githubusercontent.com/assets/3798614/9727603/dab83130-5600-11e5-911d-823edca26f6e.png)

**after**:
![red button after](https://cloud.githubusercontent.com/assets/3798614/9727602/d9fbe296-5600-11e5-9de9-4cf1d2004a2d.png)

Not sure which version I'd prefer but I tend to prefer the first one.